Emerald.App/Emerald.App/Helpers/Updater/Updater.cs

```diff
@@ -39,9 +39,7 @@ public Updater()
         bool IsInitialized = false;
         public async System.Threading.Tasks.Task Initialize()
         {
-            var cId = await FileIO.ReadTextAsync(await StorageFile.GetFileFromPathAsync($"{Windows.ApplicationModel.Package.Current.InstalledPath}\\GithubClientID.txt"));
-
-            Client = new GitHubClient(new Octokit.ProductHeaderValue(cId));
+            Client = new GitHubClient(new Octokit.ProductHeaderValue("Reverside.Emerald"));
```

Comment:
_Reverside_ 🥴

Reply:
Fixed in https://github.com/RiversideValley/Emerald/pull/36/commits/6f9843c002fe7a38d9be1b4e3e5792f6022cc3ae

Reply:
> Fixed in [6f9843c](https://github.com/RiversideValley/Emerald/commit/6f9843c002fe7a38d9be1b4e3e5792f6022cc3ae)

thanks